src/app/about/page.tsx

```diff
@@ -26,11 +27,14 @@ import { focusAreasData } from './data/focusAreasData'
 import { reportsData } from './data/reportsData'
 import { generateStructuredData } from './utils/generateStructuredData'
 
-const { header, seo } = attributes
+const aboutPageData = getPageMarkdownData({
```

Comment:
So the `aboutPageData` type has `header` and `seo`, but also the `slug`? Why do we need `slug`? I might be missing something
<img width="796" alt="Screenshot 2024-11-07 at 12 33 01" src="https://github.com/user-attachments/assets/3b9d22b3-d62b-407c-be7f-d6d2c193640d">

Reply:
We don't need `slug`, but because `getPageMarkdownData` uses `getMarkdownData` internally, it gets added to the response. I'm fine with it.

Would you prefer `getPageMarkdownData` to implement its own logic? It will look 95% similar to `getMarkdownData`, but maybe it's better this way? What do you think?

Reply:
I would probably say yes to `getPageMarkdownData` implementing its own logic.

Reply:
`getPageMarkdownData` implement its own logic now!